.github/workflows/codeql-analysis.yml

```diff
@@ -4,6 +4,11 @@ on:
   push:
     branches: [main]
   pull_request:
+    branches:
+      - main
```

Comment:
That is indeed what it means. Are you worried about PRs into release branches? We can add that type of filter to both push/pull_request as well.

Reply:
Sometimes we create PR that target other branches before merging everything to master, I guess the sooner we see a problem the better.
But this is not a big issue for me.

Reply:
Understand the concern there. In those cases we could leave a PR open for that branch into `main` and it would rerun this check on each merge of subPRs. If that isn't good enough we could also look at using a branch name filter scheme. Or we could always revisit this change.

Reply:
We can merge it and revisit it. It's such an easy change. I don't mind too much.